typescript/packages/subsurface-viewer/src/storybook/sharedSettings.tsx

```diff
@@ -98,6 +98,7 @@ export const volveWellsWithLogsLayer = {
     logrunName: "BLOCKING",
     logName: "ZONELOG",
     logColor: "Stratigraphy",
+    ZIncreasingDownwards: false,
```

Comment:
If this is missing here, it should also miss above (ie. for all the layers using ./volve_wells.json, right ?